docs/tutorials/excel-power-automate-manual.md

```diff
@@ -80,7 +80,7 @@ Power Automate shouldn't use [relative references](../testing/power-automate-tro
 
     :::image type="content" source="../images/power-automate-tutorial-4.png" alt-text="Excel Online (Business) option in Power Automate.":::
 
-7. Under **Actions**, select **Run script**.
+7. Under **Actions**, select **Run script**. This action runs a script from your OneDrive on a workbook. If you want to use a script stored in your team's SharePoint library, you'd use the **Run script from a SharePoint library** action.
```

Comment:
Grammar nit – I'm pretty sure that present tense ("want") in the if statement means the conditional clause needs "should" instead of "would". The distinction is minor in English but has an impact on the machine translation of the sentence. 
```suggestion
7. Under **Actions**, select **Run script**. This action runs a script from your OneDrive on a workbook. If you want to use a script stored in your team's SharePoint library, you should use the **Run script from a SharePoint library** action.
```